packages/ui/src/stories/Button/index.tsx

```diff
@@ -1,10 +1,10 @@
-import React from "react";
-import { ButtonStyles } from "./button.style";
+import React from 'react';
+import { ButtonStyles } from './button.style';
 
 export interface ButtonProps {
-  type: "primary" | "secondary";
+  type: 'primary' | 'secondary';
   id: string;
-  htmlType: "button" | "submit" | "reset" | undefined;
+  htmlType: 'button' | 'submit' | 'reset' | undefined;
   onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
   disabled: boolean | undefined;
   style: object;
```

Comment:
Thinking of the types again, most of them arent right here. I missed it on my initial review. Can you please make an issue for it so that we can work it out?

Reply:
I opened the issue. By the way, everything on my side is single quoted `htmlType: 'button' | 'submit' | 'reset' | undefined;`